bundles/org.openhab.binding.ws980wifi/README.md

```diff
@@ -0,0 +1,92 @@
+
+## Full Example
+
+Thing ws980wifi:ws980wifi:wetterDO `AZ WetterDO`
+    @ `Arbeitszimmer`
+    [ host=`wetterDO.fritz.box`,
+      macAddress=`00:00:00:00:00:00`,
+      port=`45000`,
+      refreshInterval=`60` ]
+
+Group    gWetterstation `Wetterstation WS980wifi` 
+Number   ws_temperatur_gefuehlt `WS Gefühlte Temp. [%.2f]°C` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:tempWindChill`}
+Number   ws_temperatur_aussen `WS Temp. außen [%.2f]°C` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:tempOutside`}
+Number   ws_temperatur_innen `WS Temp. innen [%.2f]°C` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:tempInside`}
+Number   ws_taupunkt `WS Taupunkt [%.2f]°C` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:tempDewPoint`}
+Number   ws_hitze_index `WS Hitze Index [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:heatIndex`}
+Number   ws_feuchtigkeit_innen `WS Feuchtigkeit Innen [%d]%` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:humidityInside`}
+Number   ws_feuchtigkeit_aussen `WS Feuchtigkeit außen [%d]%` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:humidityOutside`}
+Number   ws_wind_geschwindigkeit `WS Windgeschwindigkeit [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:windSpeed`}
+Number   ws_wind_boee_geschwindigkeit `WS Windgeschwindigkeit Böe[%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:windSpeedGust`}
+Number   ws_wind_richtung `WS Windrichtung [%d]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:windDirection`}
+Number   ws_luftdruck_abs `WS Luftdruck abs. [%.0f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:pressureAbsolut`}
+Number   ws_luftdruck_rel `WS Luftdruck [%.0f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:pressureRelative`}
+Number   ws_regen_altuell `WS Regen aktuell [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:rainLastHour`}
+Number   ws_regen_tag `WS Regen akt. Tag [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:rainLastDay`}
+Number   ws_regen_woche `WS Regen akt. Woche [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:rainLastWeek`}
+Number   ws_regen_monat `WS Regen akt. Monat [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:rainLastMonth`}
+Number   ws_regen_jahr `WS Regen akt. Jahr [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:rainLastYear`}
+Number   ws_regen_total `WS Regen Total [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:rainTotal`}
+Number   ws_sonne_lichtintensitaet `WS Lichtintensität [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:lightLevel`}
+Number   ws_sonne_uv_strahlung `WS UV Strahlung [%.2f]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:uvRaw`}
+Number   ws_sonne_uv_index `WS UV Index [%.d]` (gWetterstation) {channel = `ws980wifi:ws980wifi:wetterDO:uvIndex`}
+
+
+## Any custom content here!
```

Comment:
Proper chapter name missing. Acknowledgements, perhaps? I will need to check how this is usually done, and if README is the correct place for this.

Reply:
done